runtime/common/src/main/proto/ControlMessage.proto

```diff
@@ -57,6 +57,8 @@ enum MessageType {
     ExecutorFailed = 6;
     MetricMessageReceived = 7;
     ExecutorDataCollected = 8;
```

Comment:
How about pushing this value to 10, or 7 to keep the metric-related ones together?

Reply:
Addressed. Thanks!